### ISSUE TITLE
Add support for java.time package.

### ISSUE THREAD

**Issue by Argent** [google/j2objc#974]
I was trying to convert a simple test application and got this error

```
import java.time.LocalDate;

error: package java.time does not exist
```

From my understanding the java.time package is also supported by android since api level 26. Are there any plans to add this to j2objc?

**Comment by tomball** [comment]
Not all Android devices run api level 26 (Oreo); [as of last February, only around 1% do](https://en.wikipedia.org/wiki/Android_version_history) (click on the graph to expand it). Are you planning on restricting your app's potential install base to only 1% of the world's Androids? I'd double-check with whomever is handling your product marketing before committing to such a plan.

To support customers with older devices, our policy is to base j2objc's JRE on the Android libcore source that is the previous major version behind the latest Android release. That means the jre_emul is mostly based on level 25 (Nougat), which we update as bug fixes are made to that level. Since level 27 is in preview mode, we should be able to start upgrading to Oreo this year.

**Comment by Argent** [comment]
Thank you for your very fast reply.

I'm trying to port the core logic of an existing open source java application (desktop) to iOS. There are no plans for an android version.
I was asking because I read that the goal of this project is to support all java features that are available on the android platform.

With a significant amount of effort it is probably possible to refactor the current project from java.time to joda.time which is apparently convertible by j2objc. But I feel as if this would be a step backwards instead of forward.

**Comment by skovatch** [comment]
javax.time isn't Android-specific -- it's been in the JDK since [JDK 8](https://docs.oracle.com/javase/8/docs/api/java/time/package-summary.html). I'm trying to generate ObjC from some Java code that uses javax.validation annotations, and those indirectly pull in javax.time. Isn't there some level of support possible that doesn't require a full ICU4J implementation?

**Comment by tomball** [comment]
[BTW, we welcome contributions.](https://developers.google.com/j2objc/guides/how-to-contribute)

**Comment by antonio-cortes-perez** [comment]
Fixed in version 2.3.
